google_compute_engine/accounts/accounts_utils.py

```diff
@@ -150,7 +152,7 @@ def _UpdateUserGroups(self, user, groups):
     """
     groups = ','.join(groups)
     self.logger.debug('Updating user %s with groups %s.', user, groups)
-    command = ['usermod', '-G', groups, user]
+    command = self.usermod_cmd.format(user=user, groups=groups)
     try:
       subprocess.check_call(command)
```

Comment:
Given that command is now a string instead of a list, does this check_call still work?